actioncable/lib/action_cable.rb

```diff
@@ -32,7 +32,8 @@ module ActionCable
     message_types: {
       welcome: 'welcome'.freeze,
       ping: 'ping'.freeze,
-      confirmation: 'confirm_subscription'.freeze,
+      confirm_subscribe: 'confirm_subscribe'.freeze,
```

Comment:
Hmm, that's a very good question. I *think* the way the protocol stuff works is that when the client first tries to open a connection, the client + server try to negotiate and figure out which AC protocol version will be used. Thus, I *think* that that the new message types will only be picked up by `actioncable-v2-json`, which will need to be created now, since this is a breaking change.

Questions still not answered...:
- Do we need to look into how to run deprecation cycles with Action Cable protocol versions?
- How do we "drop support" for Action Cable protocol versions, given that they are versioned differently than the main Action Cable version?

If it's easier, I can not introduce a breaking change here, but for the sake of possible future breaking changes, it's probably good to figure out how to do them without breaking apps 😬 

(AC Protocol stuff was introduced via #24224, just for reference)

@jeremy and @javan, thoughts? 😬